src/main/java/de/retest/web/ScreenshotProvider.java

```diff
@@ -12,20 +18,59 @@
 
 	private static final int SCROLL_TIMEOUT = 100;
 	private static final boolean USE_DEVICE_PIXEL_RATIO = true;
+	private static final int WINDOW_WIDTH = 1200;
+	private static final int RETINA_SCALE_FACTOR = 2;
 
 	private ScreenshotProvider() {
 		// private constructor for util class
 	}
 
 	public static BufferedImage shootFullPage( final WebDriver driver ) {
+		final BufferedImage image;
 		if ( driver instanceof ChromeDriver ) {
-			return Shutterbug
+			image = Shutterbug
 					.shootPage( driver, ScrollStrategy.WHOLE_PAGE_CHROME, SCROLL_TIMEOUT, USE_DEVICE_PIXEL_RATIO )
 					.getImage();
 		} else {
-			return Shutterbug
+			image = Shutterbug
 					.shootPage( driver, ScrollStrategy.BOTH_DIRECTIONS, SCROLL_TIMEOUT, USE_DEVICE_PIXEL_RATIO )
 					.getImage();
 		}
+		if ( image.getWidth() != WINDOW_WIDTH ) {
+			int height = image.getHeight();
+			if ( isRetinaDisplay() && driver instanceof ChromeDriver ) {
+				height /= RETINA_SCALE_FACTOR;
+			}
+			return resize( image, WINDOW_WIDTH, height );
+		}
+		return image;
+	}
+
+	// TODO Remove with retest-model version 5.1.0 and use ImageUtils instead.
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Complete the task associated to this TODO comment. ([squid:S1135](https://sonarcloud.io/organizations/retest/rules#rule_key=squid%3AS1135))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=de.retest%3Arecheck-web&pullRequest=107&issues=AWcc0BUxd6oxh3R411mB&open=AWcc0BUxd6oxh3R411mB)

[ISSUE_KEY]: AWcc0BUxd6oxh3R411mB